libra_node/src/main_node.rs

```diff
@@ -200,12 +200,12 @@ pub fn setup_network(
             .into_iter()
             .map(|(peer_id, addrs)| (peer_id.try_into().expect("Invalid PeerId"), addrs))
             .collect();
-        let network_signing_private = config.peer_keypairs.take_network_signing_private()
+        let network_signing_private = config.network_keypairs.take_network_signing_private()
             .expect("Failed to move network signing private key out of NodeConfig, key not set or moved already");
         let network_signing_public: Ed25519PublicKey = (&network_signing_private).into();
         network_builder
             .transport(TransportType::TcpNoise(Some(
-                config.peer_keypairs.get_network_identity_keypair(),
+                config.network_keypairs.get_network_identity_keypair(),
```

Comment:
IIRC we shouldn't clone private keys. Don't need to handle it in this PR though.

/cc @kchalkias @huitseeker do we have any guidelines about in-memory cryptographic material? There's lots of other things we could be doing like guard pages or [clear-on-drop](https://github.com/cesarb/clear_on_drop).

Reply:
Will take care of this at a later point.